common/calibration-model.h

```diff
@@ -1,22 +1,20 @@
 // License: Apache 2.0. See LICENSE file in root directory.
-// Copyright(c) 2021 Intel Corporation. All Rights Reserved.
-
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
```

Comment:
I don't think we need to change in this PR, but for future editing - the practice should be creation date for new files, year span for edited files. i.e. 2021-2024 here.

Reply:
Next time...